.github/workflows/release_snapshot.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - 'main'
+  workflow_dispatch:
```

Comment:
Looks like this needs to be merge into main for the run workflow button to show up in github actions. Should we open a separate PR for this?

Reply:
Yeah 👍

Reply:
I just updated the workflow to trigger off of the new feature branch. As a longer term solution, we could still open up the PR adding this to main.

Reply:
I needed to add `workflow_dispatch:` back in to run the workflow.